packages/calling-component-bindings/src/handlers/createTeamsCallHandlers.ts

```diff
@@ -125,6 +127,20 @@ export const createDefaultTeamsCallingHandlers = memoizeOne(
         if (incomingCall) {
           await incomingCall.reject();
         }
+      },
+      /* @conditional-compile-remove(together-mode) */
+      onStartTogetherMode: async (): Promise<void> => {
+        if (!call) {
+          return;
+        }
+        const callState = callClient.getState().calls[call.id];
+        if (!callState) {
+          return;
+        }
+        if (!callState.togetherMode.isActive) {
```

Comment:
Is this check necessary? We should make `togetherModeFeature.start()` to be idempotent meaning if it's already started then `.start()` will noop.

That would make this function reduce down to:
```ts
onStartTogetherMode: async (): Promise<void> => {
  return call?.feature(Features.TogetherMode)?.start();
}
```

Reply:
I created a user story to look into this on the SDK side